### DISCUSSION TITLE
Possibility of .csv Export?

### DISCUSSION THREAD

**Discussion by TheZargs** [orangecoding/fredy#113] [Ideas]
Is there any way to write the found results to a .csv file to perform further calculations with the information obtained, so that everything doesn’t have to be transferred manually?

Thanks for the great application; the notification feature is a real game changer 👍👍

**Comment by orangecoding** [comment]
Hey. Sorry for the late response, somehow I don't get notified when somebody starts a discussion in here...


Can you share more details? What exactly do you want to do with the export?

A friend of mine also wanted to have the data available also after notification, so he uses the SQLite feature of Fredy to store the values and created a second process on his own which does whatever he wants :)

**Comment by TheZargs** [comment]
> A friend of mine also wanted to have the data available also after notification, so he uses the SQLite feature of Fredy to store the values and created a second process on his own which does whatever he wants :)

This is exactly what I need, where can I find the SQLite feature? I searched everywhere in the UI and found nothing. sorry if I was blind.

**Reply by orangecoding** [reply]
Hey,

sorry, I forgot my friend was using a pre-release. I just pushed the necessary changes.
Please update to the latest Release of Fredy.
Afterwards,  you can simply edit your job(s) and add the new sqlite provider. This will store the any found listing in a sqlite file within the db folder. 

See
<img width="1401" alt="Screenshot 2024-11-01 at 16 59 32" src="https://github.com/user-attachments/assets/b187c4b0-1e72-4b5b-b66a-f177b3f38ec1">

**Comment by TheZargs** [comment]
Thanks, that's exactly what I was looking for. At this point I would also like to thank you for your great project. Thank you! 🙏

**Reply by orangecoding** [reply]
Glad it helps.

🤘
